documentation/host/docker.rst

```diff
@@ -70,5 +70,18 @@ To load a configuration file into the container when starting up, we make use of
 
 .. warning:: The location of the instance folder depends on how we serve FlexMeasures. The above works with gunicorn. See the compose file for an alternative (for the FlexMeasures CLI), and you can also read the above link about the instance folder.
 
-.. note:: This is also a way to add your custom logic (as described in :ref:`plugins`) to the container. We'll document that shortly. Plugins which should be installed (e.g. by ``pip``) are a bit more difficult to support (you'd need to add `pip install` before the actual entry point). Ideas welcome. 
+Installing plugins within the container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+At this point, the FlexMeasures container is up and running without including the necessary plugins. To integrate a plugin into the container, follow these steps:
+
+1. Copy the plugin directory into your active FlexMeasures container by executing the following command:
```

Comment:
"plugin directory" is unclear. Do you mean the directory which contains the plugin's `__init__.py`?

Reply:
The main directory which has the complete code.

Reply:
I believe my description is more precise :)

Reply:
I would want to copy the complete plugin's code including requirements and not just the directory that contains `__init__.py`. By doing this, we can install either the package or it's requirements.